### PR TITLE
Hungarian translation of the essence of calculus video

### PR DESCRIPTION
**Pull request by homok43**
This translation was intended to be a quick suffix-fixing one, but instead it quickly went off the rails :D

### Problems of the AI translation

Similarly to #13, the problem of the AI translating "you" as a formal you comes up in Hungarian as well, as well the problem of out of context words and mistranslated technical concepts. However, sometimes words, or even whole sentence clauses are omitted. Many sentences range from unnatural to incomprehensible. 

Out of curiosity I tried feeding a couple strings to DeepL as proposed in #10 and it was better but wasn't quite there yet (we probably can't expect that from any tool tbh). In the end, I felt like there was something off in almost all of the sentences, so I corrected the whole thing as best as i could.

### Dev experience

It wasn't absurd to edit a raw JSON file by hand, but for broader community contribution I believe it would be better to have an interface that only allows access to the relevant data, that way more non-technical people could contribute. Some kind of proofreading or peer review/voting for submitted fixes is also something that would be great to have in my opinion.

### Questions

If these subtitles are going to be used for generating audio tracks, how should notation be transcribed in cases where writing it as you say it would negatively affect reading? (For example dA for small change in area could be pronounced as "daa" by the AI voice, but writing it any other way would cause a mismatch between the video and the subtitles). Maybe an optional pronunciation override?

### CODE REVIEW

File: 2017/essence-of-calculus/hungarian/sentence_translations.json [hunk 1]
```diff
@@ -694,7 +694,7 @@
  },
  {
   "input": "We know it gives the area under the graph of x2 between some fixed left point and some variable right point, but we don't know what it is. ",
-  "translatedText": "Tudjuk, hogy az x2 grafikonja alatti területet adja meg egy rögzített bal pont és egy változó jobb oldali pont között, de nem tudjuk, mi az. ",
+  "translatedText": "Tudjuk, hogy az x2 görbéje alatti területet adja meg egy rögzített bal pont és egy változó jobb oldali pont között, de nem tudjuk, mi az. ",
```
Comment (MrExpertGitHub) [inline 2017/essence-of-calculus/hungarian/sentence_translations.json:697]:
```suggestion
  "translatedText": "Tudjuk, hogy az x négyzet függvény görbéje alatti területet adja meg egy rögzített bal pont és egy változó jobb oldali pont között, de nem tudjuk, mi az. ",
```

**Review by MrExpertGitHub — CHANGES_REQUESTED** [review]
Kijavítottam néhány matematikai pontatlanságot
-Kaszap Máté

**Comment by 3b1b** [comment]
Good to know that DeepL is a better starting point.

As to how the TTS handles it, in a week or two we can make a few samples for you to see what it does by default, and perhaps that will inform how things should be corrected.

Is this one good to merge?

File: 2017/essence-of-calculus/hungarian/description.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 [
  {
-  "translatedText": "Milyen érzés lehet feltalálni a kalkulátort?",
+  "translatedText": "Milyen érzés lehet feltalálni a kalkulust?",
```
[the remark this answers is not in the document]
Comment (homok43) [reply 2017/essence-of-calculus/hungarian/description.json:3]: Azért maradtam a kalkulusnál, mert szerintem az analízis kicsit szigorúbbnak érződik mint ennek a sorozatnak a geometriai és infinitezimálisos megközelítése, legalábbis az elején, a 7-esnél már elég formális. Itthon se annyira ismeretlen, sok egyetemi kurzus is kalkulusként van elnevezve, meg persze ott van pl a Thomas-féle kalkulus könyv is. De igazad van, meg nekem is jobban tetszik az analízis mint kifejezés :D

File: 2017/essence-of-calculus/hungarian/sentence_translations.json [hunk 3]
```diff
@@ -37,7 +37,7 @@
  },
  {
   "input": "Calculus has a lot of rules and formulas which are often presented as things to be memorized. ",
-  "translatedText": "A Calculusnak sok szabálya és képlete van, amelyeket gyakran megjegyzendő dolgokként mutatnak be. ",
+  "translatedText": "A kalkulusnak sok szabálya és képlete van, amelyeket gyakran megjegyzendő dolgokként mutatnak be. ",
```
[the remark this answers is not in the document]
Comment (MrExpertGitHub) [reply 2017/essence-of-calculus/hungarian/sentence_translations.json:40]:
```suggestion
  "translatedText": "Az analitikának sok szabálya és képlete van, amelyeket gyakran megjegyzendő dolgokként mutatnak be. ",
```

**Review by MrExpertGitHub — CHANGES_REQUESTED** [review]

**Review by MrExpertGitHub — APPROVED** [review]

**Pull request merged**